src/ComparePage/ComparePage.tsx

```diff
@@ -0,0 +1,203 @@
+		const statsResult = await api.pingSummonerStatsEndpoint(
+			accountId,
+			numberOfGames.count
+		)
+		if (accountId == userA.accountId)
+			setUserAStats({
+				assistsAvg: statsResult.assistsAvg,
+				assistsTotal: statsResult.assistsTotal,
+				deathsAvg: statsResult.deathsAvg,
+				deathsTotal: statsResult.deathsTotal,
+				gamesCount: statsResult.gamesCount,
+				goldEarnedAvg: statsResult.goldEarnedAvg,
+				goldEarnedTotal: statsResult.goldEarnedTotal,
+				kDA: statsResult.kDA,
+				killsAvg: statsResult.killsAvg,
+				killsTotal: statsResult.killsTotal,
+				timePlayedAvg: statsResult.timePlayedAvg,
+				timePlayedTotal: statsResult.timePlayedTotal,
+				totalLosses: statsResult.totalLosses,
+				totalWins: statsResult.totalWins,
+				winPercentage: statsResult.winPercentage,
+			})
+		else if (accountId == userB.accountId) {
+			setUserBStats({
+				assistsAvg: statsResult.assistsAvg,
+				assistsTotal: statsResult.assistsTotal,
+				deathsAvg: statsResult.deathsAvg,
+				deathsTotal: statsResult.deathsTotal,
+				gamesCount: statsResult.gamesCount,
+				goldEarnedAvg: statsResult.goldEarnedAvg,
+				goldEarnedTotal: statsResult.goldEarnedTotal,
+				kDA: statsResult.kDA,
+				killsAvg: statsResult.killsAvg,
+				killsTotal: statsResult.killsTotal,
+				timePlayedAvg: statsResult.timePlayedAvg,
+				timePlayedTotal: statsResult.timePlayedTotal,
+				totalLosses: statsResult.totalLosses,
+				totalWins: statsResult.totalWins,
+				winPercentage: statsResult.winPercentage,
+			})
+		}
+	}
+	// Call fireGetStats for users A and B
+	const compareUsers = () => {
+		fireGetStats(userA.accountId)
+		fireGetStats(userB.accountId)
+	}
+	// Set page name, onload call fireGetUsers
+	useEffect(() => {
+		setTitle(`Comparison`)
+		fireGetUsers()
+		console.log('here   ' + users[0])
+		// EUREKA! This set is being called before set users happens
+	}, [])
+
+	const gamesHandleChange = (
+		event: React.ChangeEvent<{ value: unknown }>
+	) => {
+		setNumberOfGames({
+			count: event.target.value as number,
+		})
+	}
+
+	const userAHandleChange = async (
+		event: React.ChangeEvent<{ value: unknown }>
+	) => {
+		const u = users.find((user) => {
+			return user.accountId == event.target.value
+		})
+
+		if (u) setUserA(u)
+	}
+
+	const userBHandleChange = async (
+		event: React.ChangeEvent<{ value: unknown }>
+	) => {
+		const u = users.find((user) => {
+			return user.accountId == event.target.value
+		})
+
+		if (u) setUserB(u)
+	}
+
+	return (
+		// Search Section: Layout; 2 Selects for user lists, 1 Select for # of games
+		<Container>
+			{/* Search Section */}
+			<Container className={classes.searchContainer}>
+				<Typography>Games:</Typography>
+				<Select
+					onChange={gamesHandleChange}
+					value={numberOfGames.count}
+				>
+					{[
+						1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16,
+						17, 18, 19,
+					].map((num) => (
+						<MenuItem value={num}>{num}</MenuItem>
```

Comment:
Each of these elements needs a key attribute

```suggestion
						<MenuItem key={num} value={num}>{num}</MenuItem>
```